pkg/sdk/go/message.go

```diff
@@ -22,7 +22,7 @@ func (sdk mfSDK) SendMessage(chanName, msg, key string) errors.SDKError {
 		subtopicPart = fmt.Sprintf("/%s", strings.Replace(chanNameParts[1], ".", "/", -1))
 	}
 
-	url := fmt.Sprintf("%s/channels/%s/messages/%s", sdk.httpAdapterURL, chanID, subtopicPart)
+	url := fmt.Sprintf("%s/channels/%s/messages%s", sdk.httpAdapterURL, chanID, subtopicPart)
```

Comment:
In `Line#22`  leading `/` is added to subtopic ,So removed here

### Why removing here and not in `Line#22` ? 
If there is no subtopic , then ending `/` is no needed here, So better to have leading `/` in subtopics not in main topic